webapp/channels/src/packages/mattermost-redux/src/actions/posts.ts

```diff
@@ -855,10 +842,8 @@ export function getPostsSince(channelId: string, since: number, fetchThreads = t
         dispatch(batchActions([
             receivedPosts(posts),
             receivedPostsSince(posts, channelId),
-            {
-                type: PostTypes.GET_POSTS_SINCE_SUCCESS,
```

Comment:
I'm not sure why we still had this around before, but would you mind also removing this constant? I don't see it used anywhere else